modules/modules.go

```diff
@@ -0,0 +1,279 @@
+// Copyright 2015 Matthew Holt and The Caddy Authors
```

Comment:
What is the appropriate way of handling these headers? This file is a stripped down version of [the original](https://github.com/caddyserver/caddy/blob/b6e96d6f4a55f96ccbb69f112822f0a923942246/modules.go), so do we add the k6 header and leave this one below it, or can we re-license it?